SQL/0000-00-00-schema.sql

```diff
@@ -1551,10 +1551,11 @@ CREATE TABLE `certification` (
   `examinerID` int(10) unsigned NOT NULL DEFAULT '0',
   `date_cert` date DEFAULT NULL,
   `visit_label` varchar(255) DEFAULT NULL,
-  `testID` varchar(255) NOT NULL DEFAULT '',
+  `testID` int(10) UNSIGNED NOT NULL,
   `pass` enum('not_certified','in_training','certified') DEFAULT NULL,
   `comment` varchar(255) DEFAULT NULL,
-  PRIMARY KEY (`certID`,`testID`)
+  PRIMARY KEY (`certID`,`testID`),
+  CONSTRAINT `FK_certifcation` FOREIGN KEY (`testID`) REFERENCES `test_names` (`ID`) ON UPDATE CASCADE
```

Comment:
Is there any reason for the cascade? I can't think of any cases where the testID should change.